pytorch/docker-compose.yaml

```diff
@@ -187,3 +187,10 @@ services:
       - 8082:8082
       - 7070:7070
       - 7071:7071
+  hf-genai:
+    build:
+      args:
+        HF_VERSION: ${HF_VERSION:-4.41.2}
+      target: hf-genai
+    extends: multinode
+    image: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-${PACKAGE_OPTION:-pip}-py${PYTHON_VERSION:-3.10}-ipex-${IPEX_VERSION:-2.3.0}-oneccl-inc-${INC_VERSION:-2.5.1}-hf-${HF_VERSION:-4.41.2}
```

Comment:
Do we need `inc-${INC_VERSION}` in our tag? I know that we are getting INC as part of the multinode container, but we haven't been using it much with Gen AI, and I think they have more specific tools for quantization in this situation (like optimum-intel and ITREX) although we haven't experimented with those much.

Reply:
+1

Reply:
Even if you removed the whole `oneccl-inc-${INC_VERSION:-2.5.1}-` it won't bother me, but `inc` and it's version is certainly not critical here.